### PR TITLE
Add counter.set_value service

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

Adds documentation for the new `counter.set_value` service.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/92863
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/counter.markdown [hunk 1]
```diff
@@ -111,6 +111,15 @@ With this service the counter is reset to its initial value.
 | ---------------------- | -------- | ----------- |
 | `entity_id`            |      no  | Name of the entity to take action, e.g., `counter.my_custom_counter`. |
 
+### Service `counter.set_value`
+
+With this service the value of the counter to a specific value.
```
Comment (c0ffeeca7) [inline source/_integrations/counter.markdown]:
``` suggestion
With this service, set the value of the counter to a specific value.
```

Comment (c0ffeeca7) [inline source/_integrations/counter.markdown:116]:
``` suggestion
With this service, set the counter's value to a specific value.
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]

File: source/_integrations/counter.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/counter.markdown]: Hmm, don't like my original :D but not too sure about that suggestion either.

How about:

```suggestion
This service allows setting the counter to a specific value.
```

**Review by c0ffeeca7 — APPROVED** [review]

**Comment by c0ffeeca7** [comment]
LGTM, @frenck  :+1:

**Pull request merged**